### PR TITLE
Members get Newsletter: download emails to csv (commits are cleaned up)

### PR DESCRIPTION
**Pull request by weedySeaDragon**
PT Story: **Members get Newsletter: download emails to csv**
https://www.pivotaltracker.com/story/show/133081751

_Note: this replaces PR #131 .  The commits in that PR got messed up.  I've rebased and so the commits are fixed._

Dump all membership applications to .csv so the user's browser can download it.
- I also included the membership status; I made a leap and assumed it would be helpful since _all_ of the membership applications are being dumped.
- The headers and each membership_application state is translated as appropriate

Changes proposed in this pull request:
1.  added 'Export...' button on the **admin root page** (_not_ the membership_applications index) because the admin root page has additional security provided in routes
2. added 'export_ankosan_csv' action to AdminController
3. added AdminPolicy
4. added locale file entries -- **Swedish entries need translating!!!**
5. specs for AdminContoller and AdminPolicy
6. feature for export-member-email-info
7. added index on `state` for `membership_applications`

(There are more commits than there need be because I noodled around with exporting from MembershipApplication first.)

**Screenshots (Optional):**
screenshot-export-ankosan-csv-sv

screenshot-export-ankosan-csv-en

Ready for review:
@thesuss @patmbolger

### CODE REVIEW

File: config/locales/sv.yml [hunk 1]
```diff
@@ -214,18 +214,26 @@ sv:
       success: Din ansökan har lämnats in för en annan granskning.
       error: Det uppstod ett fel när du försöker att skicka in din ansökan för en annan granskning.
 
-    new_status: &status_new Nytt
+    state:
+      new: &status_new Nytt
```
Comment (thesuss) [inline config/locales/sv.yml]: Whether its "Ny" or "Nytt" depends on what specifically it is that is new. 
Ansökan ends with "n" therefore wording should be "Ny"

File: config/locales/sv.yml [hunk 2]
```diff
@@ -361,9 +369,13 @@ sv:
       org_nr: *org_nr
       manage: *manage
       state: *state
+      export: Export all applications to a CSV file
```
Comment (thesuss) [inline config/locales/sv.yml]: Exportera ansökningar som CSV

File: config/locales/sv.yml [hunk 3]
```diff
@@ -361,9 +369,13 @@ sv:
...
 
     all_applications_received: Alla inkomna ansökningar
 
+    export_ansokan_csv:
+      success: Export successful. Check your browser downloads for the file.
```
Comment (thesuss) [inline config/locales/sv.yml]: Exporten lyckades. Du hittar filen bland din webbläsares nedladdade filer.

File: config/locales/sv.yml [hunk 4]
```diff
@@ -361,9 +369,13 @@ sv:
...
+      error: There was a problem with the export.
```
Comment (thesuss) [inline config/locales/sv.yml]: Ett problem uppstod med exporten.

File: config/locales/sv.yml [hunk 5]
```diff
@@ -214,18 +214,26 @@ sv:
...
+    under_review: *status_under_review
     pending_completion: &status_pending_completion Väntar på sökande
     ask_applicant_for_info: &event_ask_applicant_for_info Behöver kompletteras
     cancel_waiting_for_applicant: &event_cancel_waiting_for_applicant Ångra Behöver kompletteras
-    waiting_for_applicant: &status_waiting_for_applicant  Väntar på sökande
+    waiting_for_applicant: *status_waiting_for_applicant
     applicant_updated_info: &event_applicant_updated_info Sökanden har kompletterat sin ansökan
-    ready_for_review: &status_ready_for_review Redo för granskning
+    ready_for_review: *status_ready_for_review
     start_review: &start_review Starta omdöme
```
Comment (thesuss) [inline config/locales/sv.yml]: Påbörja granskning

**Review by thesuss — CHANGES_REQUESTED** [review]

File: config/locales/sv.yml (hunk 5 above)
Comment (thesuss) [reply config/locales/sv.yml]: It seems you missed one?

File: config/locales/sv.yml [hunk 6]
```diff
@@ -214,18 +214,26 @@ sv:
       success: Din ansökan har lämnats in för en annan granskning.
       error: Det uppstod ett fel när du försöker att skicka in din ansökan för en annan granskning.
 
-    new_status: &status_new Nytt
+    state:
+      new: &status_new Ny
+      under_review: &status_under_review Behandlas
+      waiting_for_applicant: &status_waiting_for_applicant  Väntar på sökande
+      ready_for_review: &status_ready_for_review Påbörja granskning
```
Comment (thesuss) [inline config/locales/sv.yml]: Redo för granskning

**Review by thesuss — CHANGES_REQUESTED** [review]

File: config/locales/sv.yml (hunk 5 above)
Comment (weedySeaDragon) [reply config/locales/sv.yml]: @thesuss I've got it now.  If you agree, then can you merge this so it'll be ready to demo Sunday?

**Pull request merged**